test/functional/sqlsrv/TC43_FetchData.phpt

```diff
@@ -91,16 +105,27 @@ function checkData($col, $actual, $expected)
     return ($success);
 }
 
-if (! isWindows()) {
-    setUTF8Data(true);
+if ( !isWindows() ) {
+    setlocale(LC_ALL, "en_US.ISO-8859-1");
+}
```

Comment:
Why do you need to set it again? I thought it's already set before skipif

Reply:
I added this here as the tests needs it. Some of them have skipifs that connect some don't it's hard to tell jsut by the name. I wanted to be consistent.